azure-spring-boot/src/test/resources/aad-oauth2-complete.properties

```diff
@@ -0,0 +1,17 @@
+spring.security.oauth2.client.registration.azure.client-id=abcd
+spring.security.oauth2.client.registration.azure.client-secret=password
```

Comment:
If user want to user this starter, which property user must to provide?
As I see, the azure service url should be transparent to end user.

Reply:
The properties are following the Spring Security properties doc here: 
https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-security.html#boot-features-security-oauth2-client

It's possible to provide customized registration for azure which includes pre-configured azure oauth2 urls. 

Spring Security 5 itself provides some native customized registration in [CommonOAuth2Provider](https://github.com/spring-projects/spring-security/blob/master/config/src/main/java/org/springframework/security/config/oauth2/client/CommonOAuth2Provider.java) for `google`, `facebook` etc. 

But I wonder provide detailed README doc can already help them set up. We may consider this in the future.